contracts/lib/MerklePatriciaProof.sol

```diff
@@ -70,7 +71,7 @@ library MerklePatriciaProof {
                     }
                 }
                 //extension node ... test if means that it is empty value
```

Comment:
This comment should also be changed to make more sense. It is just an extension node that should traverse more than 0 nibbles.